src/components/Table/Table.css

```diff
@@ -77,6 +77,14 @@
 	background-color: var(--color--gray--lighter);
 }
 
+.table tr.active {
+	background-color: var(--color--blue--lighter);
+}
+
+.table td.active {
```

Comment:
@reefdog 
noted! the only reason was because when the property was only present on the row, the initial click would not show the row turning blue, only after moving the cursor would it then be highlighted. Is that ok? I think it's because the styles on the cells override the row styles while it's on hover

Reply:
perhaps something like this could work?
```css
.table tr.active {
    background-color: var(--color--blue--lighter);

    td {
        background-color: var(--color--blue--lighter);
    }
}
```

Reply:
@hminsky2002 Ah right, the hovering rule beats it.

So first of all, you're right, we want the active state to beat the normal hover state. But let's do this instead:

```css
.table .clickable:not(.active):hover {
	background-color: var(--color--gray--lighter);
}

.table .active {
	background-color: var(--color--blue--lighter);
}
```

1. This way we're explicitly saying "we want clickable rows to have a hover state, but only when they're not active" which is the right level of specificity IMO
2. No need for the element names in the selectors (e.g., `tr` or `td`), we aren't meaningfully distinguishing based on element so it's unnecessary selector specificity

Reply:
@reefdog roger that! modified rebased and force pushed!

Reply:
Oh on another note: your block above would be the first time we use SASS-like CSS nesting. I adore CSS nesting and am so glad it's finally available natively, but it's only just available in all the evergreen browsers as of December 2023. It's perfectly okay to use it, especially as we're building a "tool for the future" and don't have real users to get in trouble with yet, but let's be especially sure to test any use of it across all target browsers (Chrome, Firefox, Safari) since it's still so new.

Reply:
@reefdog yeah it was a surprise to me that it accepted the nested definition in my interpreter, I had to do a double take! I was just writing it out as pseudocode but then voila. Very happy they're making that the standard.

Reply:
@hminsky2002 It's _so wonderful_. With CSS variables, CSS nesting, `calc()`, and some of the new color methods, we're really in a golden age of native CSS and ditching preprocessed supersets like SCSS. There are still a few things they do that native CSS doesn't, but not nearly enough to justify their added build complexity IMO.